020-quarkus-http-non-application-endpoints/src/test/java/io/quarkus/qe/non_application/endpoint/CommonNonAppEndpoint.java

```diff
@@ -2,19 +2,25 @@
 
 import static io.restassured.RestAssured.given;
 
+
 import java.util.Arrays;
 import java.util.List;
 
+import org.junit.jupiter.api.Test;
+
 import io.restassured.RestAssured;
 import io.restassured.builder.RequestSpecBuilder;
 import io.restassured.specification.RequestSpecification;
 
 public abstract class CommonNonAppEndpoint {
     protected static final String ROOT_BASE_PATH = "/api/";
+    protected static final String QUARKUS_PROFILE = "quarkus.profile";
+    protected static final String NATIVE = "native";
+    protected static final boolean isNative = System.getProperty(QUARKUS_PROFILE, "").equals(NATIVE);
```

Comment:
This field name does not follow the Java convention for names. Either rename it to IS_NATIVE, or change it to be a name: isNative().